config/middleware.py

```diff
@@ -80,7 +80,7 @@ def process_template_response(self, request, response):
             ]
 
         if "feedback_survey_court" in response.context_data:
-            params["court"] = response.context_data["feedback_survey_court"]
```

Comment:
I don't love how this change to a parameter name is in a commit named "linting" -- might need better explanation. I don't understand what the middleware is trying to achieve, either.

Reply:
Why the move from court to courts?

Reply:
Reverted